cijoe/scripts/fio_prep.py

```diff
@@ -19,26 +19,37 @@ def main(args, cijoe, step):
     if not repos:
         return 1
 
-    err, _ = cijoe.run("gmake --version")
-    make = "make" if err else "gmake"
-
-    commands = [
-        (f"git clone {repos['remote']} {repos['path']} || true", "/tmp"),
-        (f"git checkout {repos['tag']} || true", repos["path"]),
-        ("git rev-parse --short HEAD || true", repos["path"]),
-        (f"{make} clean", repos["path"]),
-    ]
-
-    prefix = cijoe.config.options.get("fio", {}).get("build", {}).get("prefix", {})
-    if prefix:
-        commands += [
-            (f"./configure --prefix={prefix}", repos["path"]),
+    shell = cijoe.config.options.get("cijoe", {}).get("run", {}).get("shell", "sh")
+
+    make = "gmake" if shell == "csh" else "make"
+
+    if shell == "pwsh":
+        # 2>$null will ignore the error if repo exists
```

Comment:
If you always start by removing it, then I guess it would never exist.
What happens if you try to remove it and it doesn't exist?
Probably the `2>$null` should be moved to the `remove-item`.